docs/ci-cd-setup.md

```diff
@@ -99,11 +100,11 @@ services:
         privileged: true
         ports:
             - 3000:3000
+            - 3001:3001
         links:
             - mongo
         volumes:
-            - /home/docker/amos_files/config:/usr/src/app/dist/config
-            - /home/docker/amos_files/config.ts:/usr/src/app/config/config.ts
+            - /home/docker/amos_files/config/config.ts:/usr/src/app/shared/config/config.ts
```

Comment:
Bist du dir hier sicher, dass wir nicht beide Files benötigen? Aktuell müssen wir sowohl den config Ordner mit einer config.js und einer config.js.map, als auch die config.ts einbinden. Andernfalls stürzt Docker leider ab.

Reply:
Das müssen wir testen. In /shared/config liegt allerdings noch eine andere config für die WebSocket-Verbindung. Daher wäre es ungünstig das ganze Verzeichnis zu mounten ...
Ich teste das ganze heute Abend auf meinem Docker-Host!